src/server_utils.hpp

```diff
@@ -34,5 +34,10 @@ char customToLower(char c);
 void printFDsVector(const std::vector<pollfd> &fds);
 void print_connectionsVector(const std::vector<Connection> &connections);
 void printStrWithNonPrintables(const std::string httpRequest, size_t startPos);
+void printConnections(const std::string &location,
```

Comment:
We have quite a few prints set by overloading the << operator, wouldn't that be applicable here ?